### ISSUE TITLE
Doubts / Suggestions

### ISSUE THREAD

**Issue by RoberMS** [travelpayouts/travel-app-android#22]
hello, we have some doubts / suggestions.

1. Would it be possible for you to allow us to customize the start tabs? that is, let us initialize the search engine for flights and hotels in the activity we want. just initialize the search engine where we want,  even if it is the closed design as until now.

this would really be very simple, you would only have to put the tabs that are right now as an editable activity within the application.

This would give us the freedom to include other programs that you offer such as Discover Car Hire, Trainline, Viator etc ...

2. have you added street view in the hotels, can you put so that each one can activate or deactivate it as you wish? Street view has been paid for a while and surprises may come.

3. Could you put deep links? that is, for example to be able to pass to the application the destination and complete the search engine. or city id for hotels

**Comment by b1uebyte** [comment]
1. Not possible due to architecture of the app. A lot of changes should be done to codebase to make this work.
2. Streets view will be cut out in next release
3. Will be added in one of future releases (we have it in our roadmap list)

**Comment by RoberMS** [comment]
Okay thanks.
Please, what is urgent is to solve the error when searching for flights (code 403)

**Comment by b1uebyte** [comment]
Do you use custom white label host or default one? (https://www.travel-api.pw)

**Comment by RoberMS** [comment]
we are using a custom one, the same as in whitelabel

**Comment by zencaa** [comment]
i have te same issue regardless of travel-api.pw or our cnname whiitelabel?

**Comment by zencaa** [comment]
So at least an update to say we are working on the issue would be nice...

**Comment by b1uebyte** [comment]
@RoberMS @zencaa did it ever work for you?

**Comment by zencaa** [comment]
Yes it worked it even work on the emulators as I changed back to travel-api.pw<http://travel-api.pw>

Thinking we fixed but no



On 8 Sep 2019, at 19:28, Andrei Komrachkov <notifications@github.com<mailto:notifications@github.com>> wrote:


@RoberMS<https://github.com/RoberMS> @zencaa<https://github.com/zencaa> did it ever work for you?

**Comment by b1uebyte** [comment]
@zencaa do I understand right that it works fine on emulators, but app from google play doesn't work?

**Comment by zencaa** [comment]
That is correct and the same api is fine on iOS

The client apps can be found here
https://comparethehotel.app



On 8 Sep 2019, at 20:19, Andrei Komrachkov <notifications@github.com<mailto:notifications@github.com>> wrote:


@zencaa<https://github.com/zencaa> do I understand right that it works fine on emulators, but app from google play doesn't work?

**Comment by b1uebyte** [comment]
@zencaa I checked your apk, looks like your apk has damaged SearchParams (not 100% sure). Can you create beta release with proguard turned off (in `app/build.gradle` set `minifyEnabled false` and `shrinkResources false` )? Let me know if app will work.

**Comment by zencaa** [comment]
https://play.google.com/apps/testing/com.comparethehotel.app

**Comment by zencaa** [comment]
Same issue also many other user experiencing the same.

If it was a development issue the same issue with other users starting the same day as us would be odd...

On 9 Sep 2019, at 01:03, Andrei Komrachkov <notifications@github.com<mailto:notifications@github.com>> wrote:


@zencaa<https://github.com/zencaa> I checked your apk, looks like your apk has damaged SearchParams (not 100% sure). Can you create beta release with proguard turned off (in app/build.gradle set minifyEnabled false and shrinkResources false )? Let me know if app will work.

**Comment by b1uebyte** [comment]
@zencaa have you already added a build with proguard turned off? Latest version I see is from 7th September.

It works on ios. This lets me think that something is wrong with Android client. And at the same time I see some apps in Google Play which works fine. It looks like not so many people are involved in this issue

**Comment by zencaa** [comment]
Yep still same issue

[cid:image001.png@01D566B9.EB047B20]

**Comment by b1uebyte** [comment]
@zencaa just received updated version (v. 3) of your app. And it works for me.

**Comment by zencaa** [comment]
Thank you Andrei the 403 error is fixed having disabled pro guard...

**Comment by b1uebyte** [comment]
@zencaa which version of Android Studio did you use to build the app?

**Comment by zencaa** [comment]
3.5

**Comment by RoberMS** [comment]
Disabling proguard works correctly.

But the size of the application is even larger, please work to reduce as much as possible the size of the application.

Thanks and regards

**Comment by zencaa** [comment]
The size is smaller if you bundle instead of apk

**Comment by b1uebyte** [comment]
We will provide a fix for proguard soon

**Comment by RoberMS** [comment]
@zencaa We work with Bundle, but 34MB is not a small application.

@b1uebyte Ok thanks!

**Comment by zencaa** [comment]
@RoberMS 7mb more than before but a least flights work.
